src/components/AdjustVaultSummary.tsx

```diff
@@ -80,7 +80,16 @@ const AdjustVaultSummary = () => {
   } = vaultToAdjust;
 
   const isActive = vaultState === 'active';
-  const offerButtonLabel = isActive ? 'Make Offer' : vaultState;
+
+  const offerSigner = useAtomValue(offerSignerAtom);
+
+  const offerButtonLabel = useMemo(() => {
+    if (!offerSigner?.isDappApproved) {
+      return 'Enable Dapp in Wallet';
```

Comment:
is this what clicking the button will do? if not the button should be disabled and the user instruction go somewhere else.

Reply:
No it's more of a help tip and the button should be disabled in this case. Just added an assert to make that clear.